### ISSUE TITLE
border_width works only for first monitor

### ISSUE THREAD

**Issue by kazufukurou** [baskerville/bspwm#377]
I have 2 monitor setup. Second monitor have 1 pixel border but correct gaps. When second line is uncommented then it works.

bspwmrc
```
bspc config border_width 3
#bspc config -m next border_width 3
bspc config window_gap 6
```
```
$ bspwm -v
0.9-126-gb1d9653
```

**Comment by neeasade** [comment]
I believe this is intentional -- that is, you target one monitor with this, by default the focused monitor gets the change, and then you switch to the next one to change there, etc.

**Comment by baskerville** [comment]
The behavior of those settings was fixed in ebad909.
